custom-connectors/EleadSalesOpportunities/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3912 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Elead Sales Opportunities",
+    "version": "1.0",
+    "description": "Elead Sales Opportunities API searches for and retrieves existing sales opportunity information in the Elead CRM. Opportunity information includes details related to the sales team, source, vehicle of interest and vehicle trade information.This API includes the following capabilities and features for managing opportunity-related information.",
+    "contact": {
+      "name": "CDK Global",
+      "url": "https://serviceconnect.support.cdk.com",
+      "email": "mbd_support@cdk.com"
+    }
+  },
+  "x-ms-connector-metadata": [
+      {
+        "propertyName": "Website",
+        "propertyValue": "https://fortellis.io"
```

Comment:
@amancdk Can you please share a web page or location from your company web site where I can find some information about this service or connector.

Reply:
Hi @sriyen-msft , Added details in readme.md about the details. More details can be found at - https://apidocs.fortellis.io/specs/3c6d6b40-7215-4a81-a087-4f956bb36ec9